event-management-frontend/src/components/userMapEvents/UserMapDisplayLocationsDumb.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import { Marker, Popup } from 'react-leaflet';
+import { blackMarkerPoint } from './userMarkerPointIcons';
+import { useHistory } from 'react-router-dom';
+import { EventCardsWithLoction } from './UserMapEventsSmart';
+import { useUserMapEvents } from '../../styles/userMapEvents/UserMapEventsStyle';
+
+interface Props {
+  submitLocation: (lat: string, long: string) => void;
+  eventCards: EventCardsWithLoction[];
+}
+
+const UserMapDisplayLocationsDumb = (props: Props) => {
+  const classesMap = useUserMapEvents();
+  const history = useHistory();
+  const goToEventDetails = (eventId: number) => {
+    history.push(`/user/events/${eventId}`);
+  };
+
+  return (
+    <div>
+      {props.eventCards.map((e) => (
+        <Marker
+          onClick={() => {
+            return props.submitLocation(e.location.latitude, e.location.longitude);
```

Comment:
In cases like this, you can shorten it to onClick={() => props.submitLocation(bla...)}. I.e., if there is only one instruction, you can avoid using paranthesis and return.

Reply:
resolved all fixes